src/StreamFactoryInterface.php

```diff
@@ -9,15 +9,11 @@
     /**
      * Create a new stream from a resource or a string.
      *
-     * If the argument is a resource, it MUST be readable and SHOULD be seekable. It MAY be writable.
-     * If the argument is a string, a temporary resource will be created that is writable and seekable.
-     * 
-     * If the arugment is a string it will be interpided as the content of the stream. File names or 
-     * file paths are not supported. 
+     * String content will be used as the body of the stream. Files paths are not supported.
```

Comment:
```php
public function createStream($resource_or_path, $mode = "r");
```

This is not the signature I suggested and, as I showed previously, there is no benefit to allowing file paths for `createStream()`. If we don't support file paths, there is very little reason to have separate interfaces. Or am I missing something?